src/components/Collapsible/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import { Flipped, Flipper } from 'react-flip-toolkit';
+import { Props } from './types';
+
+
+/*
+ ** Collapsible component uses CSS transitions to smoothly hide and show a block of content.
+ */
+function Collapsible({
+    className,
+    anchorContent,
+    isCollapsed = true,
+    children,
+    ...otherProps
+}: Props): ReactElement {
+
+
+    const [collapsed, setCollapsed] = React.useState(isCollapsed);
```

Comment:
Do you not need to watch isCollapsed in useEffect? I had to do this in my component in order for it to work in storybook. 

```
React.useEffect(() => {
    setCollapsed(isCollapsed);
}, [isCollapsed]);
```

Reply:
Probably not since there is always something to click on, which brings me to think my useEffect is not necessary in real practice, but necessary for storybook?